requirements.txt

```diff
@@ -26,6 +26,8 @@
 -e git+https://github.com/inspirehep/invenio-query-parser.git@invenio3-inspire#egg=invenio-query-parser==0.6.0
 -e git+https://github.com/inspirehep/invenio-pidstore.git#egg=invenio-pidstore
 
+-e git+https://github.com/inveniosoftware/invenio-collections.git@master#egg=invenio-collections==1.0.0a3.dev20160204
+
```

Comment:
Because the there has not been a release since the last commit (the one that makes `invenio-collections` work with Invenio 3)